templates/dockstore.model.ts.template

```diff
@@ -42,8 +43,7 @@ export class Dockstore {
   static readonly CWL_VISUALIZER_URI = 'https://view.commonwl.org';
 
   static readonly FEATURES = {
-    enableCwlViewer: {{ ENABLE_CWL_VIEWER }},
-    enableLaunchWithFireCloud: {{ ENABLE_LAUNCH_WITH_FIRECLOUD }}
```

Comment:
Is this just always-on now?

Reply:
Yes, I had removed it from dockstore.model.ts in the ui2 repo a while ago, but never updated it here. When adding Terra, I realized I should bring it into sync (but forgot to add a comment about it my commit message -- will do so when I merge)